examples/spark/zeppelin-controller.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
         - name: zeppelin
-          image: gcr.io/google_containers/zeppelin:v0.5.5_v1
+          image: gcr.io/google_containers/zeppelin:v0.5.5_v2
```

Comment:
It's similar to how the Spark containers were tagged originally so I followed it. Is there a better scheme for "given semver vX.Y.Z[-foo], this is docker image N?" We should probably standardize on something.